frontend_lib/src/container/RowWrappers/ListItemRowWrapper.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+
+import { PAGE } from '../../helper.js'
+import ListItemWrapper from '../../component/Lists/ListItemWrapper/ListItemWrapper.jsx'
+
+const ListItemRowWrapper = (props) => {
+  const contentTypeInfo = props.contentType.find(info => info.slug === props.originalType)
+  const contentAppUrl = props.content ? PAGE.WORKSPACE.CONTENT(
+    props.content.workspaceId,
+    contentTypeInfo.slug,
+    props.content.id
+  ) : undefined
+
+  return (
+    <ListItemWrapper
+      label={props.originalLabel}
+      read
```

Comment:
This comment is really just a suggestion, you can skip it and directly resolve this conversation if you want.
For now this component is used only for favorites, but if we think of a more generic component, I think it would be logical to add a props for read. With:
```
read={props.isRead}

[...]

isRead: PropTypes.bool

[...]

isRead: true
```